src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  const BIG_DISCOUNT = 50;
+  const SMALL_DISCOUNT = 20;
+
+  const PERIOD_WITHOUT_DISCOUNT = 2;
+  const PERIOD_WITH_SMALL_DISCOUNT = 7;
```

Comment:
The constant 'PERIOD_WITH_SMALL_DISCOUNT' is misleading because it actually represents the threshold for the 'BIG_DISCOUNT'. Consider renaming it to 'LONG_TERM_RENTAL_THRESHOLD' for clarity.